iohub/ngff/nodes.py

```diff
@@ -464,9 +462,11 @@ def get_tile_slice(
                     f"got type {type(pre_dims)}."
                 )
             for i, sel in enumerate(pre_dims):
+                if isinstance(sel, int):
+                    sel = slice(sel)
                 if sel is not None:
                     pad[i] = sel
-        return tuple(pad) + (r_slice, c_slice)
+        return tuple((pad + [r_slice, c_slice]))
```

Comment:
```suggestion
        return tuple(*pad, r_slice, c_slice)
```
Personal preference, this suggestion could be ignored.

Reply:
I was trying to appeal pyright, as is the case with type checkers, it complains about your totally valid suggestion...